plugins/blockvault_client_plugin/tests/test.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+CONTAINER_ID=`docker run  -p 5432:5432 -e POSTGRES_PASSWORD=password -d postgres`
```

Comment:
This exposes 5432 to the world. Unless that's needed it's probably better practice to do `-p  127.0.0.1:5432:5432` instead